### DISCUSSION TITLE
Filament Livestreams

### DISCUSSION THREAD

**Discussion by dissto** [filamentphp/filament#4605] [General]
Is there currently any livestream planned? I recently stumbled upon past broadcasts, and I found them very interesting to watch.
They give a nice insight and are generally nice to watch. (I watched them as a VOD though, so I haven't had the pleasure of interacting with you guys.)

If anybody wonders what livestreams I mean: [https://www.youtube.com/c/ryangjchandler/videos](https://www.youtube.com/c/ryangjchandler/videos)

@danharrin  @ryangjchandler

**Comment by danharrin** [comment]
I'd be up for livestreaming some v3 development or something!

**Reply by dissto** [reply]
Yes please 🙏

**Comment by ryangjchandler** [comment]
I've been non-stop with work for a couple of months but will definitely be livestreaming again soon!

**Comment by zepfietje** [comment]
Love to jump in as a Ryan-replacement if he's too busy to join Dan on a v3 dev stream! 😜

**Reply by danharrin** [reply]
I don't have any setup for streaming so i'd need to set that up

**Reply by zepfietje** [reply]
Oh so we do need Ryan, hehe. But we should be able to get that set up. 👍

**Comment by dissto** [comment]
Are there any news? Are livestreams planned? 🙏 🤞

**Reply by danharrin** [reply]
Not at the moment, the whole team is super busy with this release

**Comment by dissto** [comment]
Still having my hopes up for some first party Livestreams 🤞😁

**Reply by danharrin** [reply]
@alexandersix

**Reply by alexandersix** [reply]
We _are_ looking at doing some first-party livestreams as part of our content calendar (multiple Filament team members have done streaming before), but it isn't at the top of our priorities quite yet. They will come, but we have some other content pillars to reinforce first!

Don't worry, though, I love livestreams, so you can bet I won't forget about them any time soon–my Filament livestreams are a big part of why I'm on the team now!

**Reply by dissto** [reply]
It has been 6 months. Something on the horizon? 🤓 Unfortunately, there is hardly any english content (video/streams) regarding filament ☹️

**Reply by alexandersix** [reply]
We are still focusing in other areas away from livestreams at the moment. More content is coming soon, but livestreams are generally lower on the priorities list than things like articles and planned videos.
